lib/bundler/resolver.rb

```diff
@@ -131,7 +131,7 @@ def source
       def for?(platform, required_ruby_version)
         if spec = @specs[platform]
           if required_ruby_version && spec.respond_to?(:required_ruby_version) && spec_required_ruby_version = spec.required_ruby_version
-            spec_required_ruby_version.satisfied_by?(required_ruby_version.gem_version)
+            spec_required_ruby_version.satisfied_by?(Gem::Version.new("#{required_ruby_version.gem_version}.#{required_ruby_version.patchlevel}"))
```

Comment:
Thanks. I tried changing `Bundler::RubyVersion#initialize` to always include the patch version in `@gem_version`, but some tests started failing. From your experience, do you think it would be better to do this or handle this specific case in `#for?` ?

Maybe in some parts of bundler it is preferable not to have the patch version?